02-technical-guides/directquery-intro/main.ipynb

```diff
@@ -13,13 +13,12 @@
    "id": "a671e16d-c06b-432f-92c6-5b2d2f7043bd",
```

Comment:
<p>"We can only use store aggregated measures supporting disaggregation can be used in aggregate provider, e.g. <code>tt.agg.sum</code> or <code>tt.agg.mean</code> on table columns." &lt;-- I have no idea what this is trying to say 🤔, thoughts?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/865/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Reply:
Spoke to @printhellohetal and we can revise this another time! No need to block this PR for this one line.